.travis.yml

```diff
@@ -33,7 +33,7 @@ before_install:
 script: ERLC_OPTS="$ERLC_OPTS" make compile
 
 after_success:
-  - if [[ 0 -ne `git status --porcelain -- applications/ core/ deps/ | wc -l` ]]; then echo Unstaged changes! && git status --porcelain && exit 1; fi
```

Comment:
Actually no veto. That should have been removed after `.gitignore` sanitization.